### PR TITLE
Color/Typography

### PR DESCRIPTION
**Pull request by MBryder**
The text size has been updated from 12px to 14px, and from 16px to 18px. 

Additionally, the primary color has been changed to #166276. However, we renamed the 'primary' color to 'primary_custom' due to Tailwind's naming conventions, which apply a prefix to the 'primary' name

### CODE REVIEW

**Comment by Lowkee1g** [comment]
Could you please change the base branch for this pull request from ‘main’ to ‘Dev’?

**Review by Lowkee1g — APPROVED** [review]

File: eml/package-lock.json
Comment (Andersraae) [inline eml/package-lock.json:1]: This smells like problems. What was added? Is it needed?

Comment (MBryder) [reply eml/package-lock.json:1]: In what part?

Comment (MBryder) [reply eml/package-lock.json:1]: @Lowkee1g did it

Comment (Andersraae) [reply eml/package-lock.json:1]: Nvm I tested it again. No clue what is going on

Comment (Lowkee1g) [reply eml/package-lock.json:1]: > @Lowkee1g did it
🧢

**Pull request merged**